.gitignore

```diff
@@ -134,3 +134,9 @@ venv.bak/
 env3/
 
 *.bak
+
+#Pipfiles
+Pipfile*
+
+#vscode-settings
+.vscode/
```

Comment:
we have VS Code on L17 -- we should group at the very least. Or we could try to use `*` to condense ignore into one line

Reply:
Just used * to condense ignore into one line